tasks/deploy.yaml

```diff
@@ -3,11 +3,24 @@ tasks:
     inputs:
       options:
         description: For setting deploy time variables and flags
+      path:
+        description: Path relative to the repositories root where the uds-bundle.yaml lives
+        default: bundle
+      config:
+        description: File name of the bundle config file. Defaults to uds-config.yaml. Path is relative to the path input.
+        default: uds-config.yaml
     actions:
       - description: Deploy the UDS Zarf Package
-        cmd: uds zarf package deploy zarf-package-*-${UDS_ARCH}-*.tar.zst --confirm --no-progress ${{ .inputs.options }}
+        cmd: UDS_CONFIG=${{ .inputs.path }}/${{ .inputs.config }} uds zarf package deploy ${{ .inputs.path }}/zarf-package-*-${UDS_ARCH}-*.tar.zst --confirm --no-progress ${{ .inputs.options }}
```

Comment:
I dont think this will work. Zarf (whether standalone or called from uds) has its own config file. I dont think we need this here, maybe in the future, but we dont have. one by default and as implemented this actually wont work.

Reply:
I have removed it.